command/results_gotest_parser.go

```diff
@@ -30,6 +30,9 @@ var (
 
 	// Match the end prefix, save PASS/FAIL/SKIP, save the decimal value for number of seconds
 	gocheckEndRegex = regexp.MustCompile(`(PASS|SKIP|FAIL): .*.go:[0-9]+: (\S+)\s*([0-9\.m]+[ ]*s)?`)
+
+	// Match the failing status prefix for go build.
+	goTestFailedStatusRegex = regexp.MustCompile(`FAIL[^:]\s+(\S+) .*`)
```

Comment:
What's the [^:] part for? Having it here makes the example given in the ticket not match this regex I think

Reply:
No, the [^:] means that it doesn't have a colon (i.e. how test results are formatted). The example in this ticket is ` FAIL	go.mongodb.org/mongo-driver/mongo/integration [build failed]`

Reply:
But actually I can remove this and just use \s+ since that's effectively the same logic.

Reply:
Edit: I can't remove this because then other regexps also match, but I will fix it because actually I realized with \s+ it doesn't work with only one space.